packages/terra-consumer-layout/src/Layout.scss

```diff
@@ -27,8 +28,56 @@
     min-height: 100vh;
   }
 
+  .nav {
```

Comment:
what's the consideration of moving nav css into layout style?

Reply:
lets chat in person monday. @mhemesath explained it to me. probably easier in person than on github.